dev/releases/README.md

```diff
@@ -1,12 +1,36 @@
-# How To Make a GAP Release for version Z.X.Y
+# How to release GAP version X.Y.Z
  
-**This release process assumes the following**
-- A stable branch of GAP has been identified or agreed upon. This branch will be identified here as `stable-Z.X.Y` and the release notes in `CHANGES.md` have been created.
-- You run these scripts in the root directory of a clone of the [GAP repository](https://github.com/gap-system/gap)
-- You have compiled this clone of GAP
-- You have a clone of the [GapWWW repository](https://github.com/gap-system/GapWWW)
+**These instructions assume the following**
+- It has been agreed to make a release called `X.Y.Z` from the `stable-X.Y` branch.
+	- Note that most likely `X=4` for the foreseeable future.
+- All changes to be included in the release are now present in the `stable-X.Y` branch.
+	- This includes the release notes for version `X.Y.Z` being included in `CHANGES.md`.
 
-## Dependencies
+
+## The release process for GAP -- the quick guide
+
+This is the way the process should happen in practice, with the help of GitHub Actions. Instructions for achieving the same without GitHub Actions are given later (in case GitHub Actions breaks or disappears).
+
+1. Access your local clone of the GAP repository.
+2. Make sure that your clone is up to date with `gap-system/gap`.
+3. Create an annotated tag `vX.Y.Z` in your clone at the appropriate commit in the `stable-X.Y` branch, and push the tag to `gap-system/gap`. For example:
+    ```
+    git tag -m "Version X.Y.Z" vX.Y.Z stable-X.Y
+    git push origin vX.Y.Z
+    ```  
+4. Wait. Pushing the tag will trigger the “[Wrap releases](https://github.com/gap-system/gap/actions/workflows/release.yml)” GitHub Actions workflow on `gap-system/gap`, which wraps the release archives and Windows installers, and creates a release on GitHub with the archives and installers attached. This takes around 90 minutes.
+5. Once the “Wrap releases” workflow is finished, either manually dispatch the “[Sync](https://github.com/gap-system/GapWWW/actions/workflows/sync.yml)” GitHub Actions workflow on `gap-system/GapWWW`, or wait overnight for it to happen on schedule.
+	- This Workflow creates a pull request to `gap-system/GapWWW`, which updates the GAP website according to the release data hosted on `gap-system/gap`.  Check that the result is sensible.
+6. When it is time to publicise the new GAP release, merge the pull request to GapWWW.
```

Comment:
I think this is missing one step: SSH'ing into the webserver in St Andrews and (a) installing the latest package manuals there and (b) `git pull`ing the latest GapWWW changes.

Granted, this seems to be missing in the existing README as well, but we should add it (on the long run, we should of course ideally get rid of that step / automate it too, but AFAIK this is not the case yet, is it?)

Reply:
Correct, the steps of:
* extracting the GAP and package manuals with GapWWW's `etc/extract-manuals.py` script
* putting them in the right place on the server
* updating the git repository on the server
* building the new Jekyll website

still need to be done manually. I'll add at least some information about that in this PR.

Reply:
(I also agree that we want automate these steps as much as possible 🙂)

Reply:
(I also agree that we want automate these steps as much as possible 🙂)